### PR TITLE
Add result comment for docker tests

### PR DESCRIPTION
**Pull request by LongyuZhang**
- Add result comment for docker tests to correct TKG summary
- Related Issue: ibm_git/runtimes/backlog/issues/1056

### CODE REVIEW

**Comment by LongyuZhang** [comment]
Test Link: hyc_grinder 32426

File: external/dockerfile_functions.sh [hunk 1]
```diff
@@ -578,6 +582,7 @@ generate_dockerfile() {
     print_legal ${file};
     print_adopt_test ${file} ${test};
     print_image_args ${file} ${os} ${version} ${vm} ${package} ${build};
+    print_result_comment_arg ${file};
```
Comment (llxia) [inline external/dockerfile_functions.sh:585]: print_result_comment_arg() does not take any parameter

Comment (llxia) [reply external/dockerfile_functions.sh:585]: sorry, my mistake :)

**Review by llxia — APPROVED** [review]
LGTM

**Review by renfeiw — APPROVED** [review]
LGTM

**Pull request merged**